src/components/Layout/Header/AutoCompleteSearch/AutoCompleteSearch.tsx

```diff
@@ -57,7 +57,7 @@ export const AutoCompleteSearch = ({ filterBy }: AssetSearchProps) => {
         onSubmit={(e: FormEvent<unknown>) => e.preventDefault()}
       >
         <InputGroup size='lg'>
-          <InputLeftElement pointerEvents='none'>
+          <InputLeftElement pointerEvents='none' zIndex={1}>
```

Comment:
I seen you added some zIndex, why? It looks like they are not needed at all

Reply:
Yeah, so this was in response to an issue raised in the [demo wallet enhancements thread](https://discord.com/channels/554694662431178782/973365713437884436)
Turns out that it's not actually related to these changes, and the fix could have been its own PR, but it was only two lines, so I snuck it in.
I'll add a comment to those changes for clarity